src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -541,12 +541,12 @@ private void reconcileHypervisorData(
                   system.addBucket(bucket);
                   seenBucketKeys.add(bucket.getKey());
                 });
-        // remove any buckets for guests no longer present
-        system
-            .getBuckets()
-            .removeIf(b -> b.getKey().getAsHypervisor() && !seenBucketKeys.contains(b.getKey()));
       }
     }
+    // remove any buckets for guests no longer present
```

Comment:
Would you mind just adding a note here about what `asHypervisor` is meant to represent here?  I've been looking at this code for a few days and my understanding is still murky.  In 2 weeks time, I'll never remember :smile:

Reply:
Done, feel free to reopen if you'd like me to clarify further.

Reply:
Perfect.  Now we just need to change the column name so that it makes sense. :laughing: